src/networking/index.ts

```diff
@@ -28,22 +50,22 @@ export default class Client {
     this.timeout = timeout;
   }
 
-  post(path, body) {
+  post(path: string, body: any) {
```

Comment:
this should be able to use `unknown`, but might needs to replace the type of the body argument on `request()` to unknown as well:

```suggestion
  post(path: string, body: unknown) {
```

Same could work for the other methods here.